main.js

```diff
@@ -0,0 +1,239 @@
+/////////////////////////////////////////////////variables///////////////////////////////////////////////
+let square0 = document.getElementById("0")
+let square1 = document.getElementById("1")
+let square2 = document.getElementById("2")
+let square3 = document.getElementById("3")
+let square4 = document.getElementById("4")
+let square5 = document.getElementById("5")
+let square6 = document.getElementById("6")
+let square7 = document.getElementById("7")
+let square8 = document.getElementById("8")
```

Comment:
You could save some lines here and create your array using the querySelectorAll method and Array.from() like this:
```js
let squareNodeList = document.querySelectorAll(".square")
let squareArray = Array.from(squareNodeList)
```
or alternatively you could use .getElementsByClassName()